scrapi/base/__init__.py

```diff
@@ -89,16 +82,13 @@ def get_records(self, url, start_date, resump_token=''):
         print url
         data = requests.get(url)
 
-        try:
-            doc = etree.XML(data.content)
-        except etree.XMLSyntaxError:
-            return 'try a new url'
+        doc = etree.XML(data.content)
```

Comment:
Yep, error no longer occurs - handled it in a better way by just accepting longer URLs if necessary!